mu-plugins/run.sh

```diff
@@ -4,6 +4,7 @@
 
 if [ ! -d /shared/.git ]; then
   rsync -a --delete --delete-delay /mu-plugins/ /shared/
+  rsync -r --exclude-from="/mu-plugins-ext/.dockerignore" /mu-plugins-ext/* /shared
```

Comment:
Should this also have a`--delete` flag? 
```suggestion
  rsync -r --delete --exclude-from="/mu-plugins-ext/.dockerignore" /mu-plugins-ext/* /shared
```